test/test_quantization.py

```diff
@@ -311,6 +311,26 @@ def checkQuantized(model):
 
         checkQuantized(model)
 
+    def test_train_save_load_eval(self):
+        model = AnnotatedTwoLayerLinearModel()
```

Comment:
Suggest sweeping over multiple qconfigs here (cover both per tensor and per channel quant)